lib/defra_ruby/aws/services/bucket_loader_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module DefraRuby
+  module Aws
+    class BucketLoaderService
+      def self.run(bucket, file)
+        new(bucket, file).run
+      end
+
+      def initialize(bucket, file)
+        @bucket = bucket
+        @file = file
+      end
+
+      def run
+        s3_bucket.object(File.basename(file.path)).upload_file(file.path, server_side_encryption: :AES256)
```

Comment:
This is a temporary code. Will wrap it in a Response object that will deal with failures in a subsequent PR